packages/wrangler/package.json

```diff
@@ -98,11 +98,11 @@
 	},
 	"dependencies": {
 		"@cloudflare/kv-asset-handler": "^0.2.0",
-		"@esbuild-plugins/node-globals-polyfill": "^0.1.1",
-		"@esbuild-plugins/node-modules-polyfill": "^0.1.4",
+		"@esbuild-plugins/node-globals-polyfill": "^0.2.3",
```

Comment:
See https://github.com/cloudflare/workers-sdk/pull/3832

Reply:
This bump is needed with the esbuild bump, once these changes go in we can consider changing this dependency?